packages/hardhat/hardhat.config.ts

```diff
@@ -67,6 +67,20 @@ const config: HardhatUserConfig = {
       url: `https://polygon-mumbai.g.alchemy.com/v2/${providerApiKey}`,
       accounts: [deployerPrivateKey],
     },
+    gnosis: {
+      url: `https://rpc.ap-southeast-1.gateway.fm/v4/gnosis/non-archival/mainnet`,
+      httpHeaders: {
+        authorization: `Bearer ${process.env.GATEWAY_API_KEY}`,
+      },
+      accounts: [deployerPrivateKey],
+    },
+    gnosisChiado: {
+      url: `https://rpc.eu-central-2.gateway.fm/v4/gnosis/archival/chiado`,
+      httpHeaders: {
+        authorization: `Bearer ${process.env.GATEWAY_API_KEY}`,
+      },
+      accounts: [deployerPrivateKey],
```

Comment:
I'd use the official RPCs listed on https://docs.gnosischain.com/tools/rpc/#gnosis

https://rpc.gnosischain.com && https://rpc.chiadochain.net

Reply:
FYI - Gnosis will be sunsetting the "official" RPCs in favor of an ecosystem approach. Gateway will be taking over the "public" RPC. 

https://forum.gnosis.io/t/gip-70-should-gnosisdao-pass-a-grant-proposal-to-partner-with-gateway-fm/6079